tests/test_common.py

```diff
@@ -981,3 +982,27 @@ def test_cell_bounds(grid):
 def test_cell_type(grid):
     ctype = grid.cell_type(0)
     assert isinstance(ctype, int)
+
+
+def test_serialize_deserialize(grid):
```

Comment:
I'd like to see a parameterized test that serializes and deserializes a bunch of pyvista objects.  Ideally, the big five:
- `PolyData`
- `UnstructuredGrid`
- `UniformGrid`
- `StrucutredGrid`
- `RectilinearGrid`

For reference, see:
https://github.com/pyvista/pyvista/blob/87a2003ad828b0b8773e95caf7ada43bb315d447/tests/test_filters.py#L11-L17

Reply:
Thanks, that is a useful snippet. This is implemented now.